### PR TITLE
kadnode: update to version 2.2.5

### PR DESCRIPTION
**Pull request by mwarning**
Signed-off-by: Moritz Warning <moritzwarning@web.de>

Maintainer: me
Compile tested: OpenWrt master
Run tested: ramips-mt7620-wt3020, program starts

### CODE REVIEW

File: net/kadnode/Makefile [hunk 1]
```diff
@@ -2,14 +2,14 @@ include $(TOPDIR)/rules.mk
 
 
 PKG_NAME:=kadnode
-PKG_VERSION:=2.2.3
-PKG_RELEASE:=1
+PKG_VERSION:=2.2.5
+PKG_RELEASE:=0
```
Comment (karlp) [inline net/kadnode/Makefile]: just leave it at 1.  you only need to change this from 1 if you make packaging changes without changing the version.

Comment (mwarning) [reply net/kadnode/Makefile]: Isn't it set back to 0 on every release?

Comment (karlp) [reply net/kadnode/Makefile]: You'd be the first... :)

```
$ grep -r PKG_RELEASE * | cut -d: -f2,3 | sort | uniq -c | sort -gr | head
    557 PKG_RELEASE:=1
    142 PKG_RELEASE:=2
     40 PKG_RELEASE:=3
     24 PKG_RELEASE:=4
     15 PKG_RELEASE:=5
     14 PKG_RELEASE:=6
     13 PKG_RELEASE=$(PKG_SOURCE_VERSION)
     13 PKG_RELEASE=1
      5 PKG_RELEASE:=8
```

It goes back to 1.  See also https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Comment (mwarning) [reply net/kadnode/Makefile]: Convinced :-)

**Comment by karlp** [comment]
merged in 893851783 thanks

**Pull request closed without merging**